<launch> 
    <node name="ecg" pkg="ecg" type="ecg" output="screen" />

    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- markov chain -->
    <param name="state0" value="78,19,2,1,0" />
    <param name="state1" value="8,78,13,1,0" />
    <param name="state2" value="1,15,71,12,1" />
    <param name="state3" value="1,2,16,74,7" />
    <param name="state4" value="1,1,2,22,73" />

    <!-- risk values -->
    <param name="HighRisk0" value="0,69.99" />
    <param name="MidRisk0" value="70,84.99" />
    <param name="LowRisk" value="85,96.99" />
    <param name="MidRisk1" value="97,114.99" />
    <param name="HighRisk1" value="115,300" />

    <!-- accuracy in percentage -->
    <param name="data_accuracy" value="90" type="double" />
    <param name="comm_accuracy" value="100" type="double" />

    <!-- persistence -->
    <param name="persist" value="true" type="bool" />
    <param name="path" value="ecg_output.csv" type="str" />

</launch>